lib/afi.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'open-uri'
+require 'nokogiri'
+require_relative './sequel.rb'
+
+module AmericanFilmInstitute
+  def self.scrap_ranking
+    url = 'https://www.afi.com/afis-100-years-100-movies-10th-anniversary-edition/'
+    html = URI.open(url)
+    doc = Nokogiri.HTML(html)
+    movies_list = doc.css('.q_title')[0...100]
+
+    movies_list.each do |movie|
+      ranking, _, title, _, year = movie.content.match(/(\d+)(\.\s+)([^\(]+\S)(\s+\()(\d{4})(\)$)/).captures
+  
+      tmdb_movie = Tmdb.search_movie(title, year)
+      movie_record = Movie.find(imdb_id: tmdb_movie['imdb_id'])
+
+      puts title + ' is ranked #' + ranking + ' on AFI 100 list'
```

Comment:
What about interpolation?

```suggestion
      puts "#{title} is ranked ##{ranking} on AFI 100 list"
```

Reply:
Don't know what I was thinking ^^"